wasm/src/lib.rs

```diff
@@ -1,28 +1,36 @@
+mod wasm_builtins;
+
 extern crate rustpython_vm;
 extern crate wasm_bindgen;
-use rustpython_vm::compile;
+extern crate web_sys;
+
 use rustpython_vm::VirtualMachine;
+use rustpython_vm::builtins;
+use rustpython_vm::compile;
+use rustpython_vm::pyobject::PyContext;
+use std::collections::HashMap;
 use wasm_bindgen::prelude::*;
-
-#[wasm_bindgen]
-extern "C" {
-    // Use `js_namespace` here to bind `console.log(..)` instead of just
-    // `log(..)`
-    #[wasm_bindgen(js_namespace = console)]
-    fn log(s: &str);
-}
+use web_sys::console;
 
 #[wasm_bindgen]
 pub fn run_code(source: &str) -> () {
     //add hash in here
-    log("Running RustPython");
-    log(&source.to_string());
-    let mut vm = VirtualMachine::new();
+    console::log_1(&"Running RustPython".into());
+    console::log_1(&"Running code:".into());
+    console::log_1(&source.to_string().into());
+    console::log_1(&"----- console -----".into());
+
+    let ctx = PyContext::new();
```

Comment:
It would be good if the virtual machine would remain instantiated between different calls to run_code so that we could preserve local variables.

Reply:
I'll need to study how to do this in WASM. I'll create a followup issue for that.